cranelift-codegen/src/ir/dfg.rs

```diff
@@ -660,6 +660,17 @@ impl DataFlowGraph {
         self.insts[inst].put_value_list(branch_values)
     }
 
+    /// Append new value arguments to an instruction.
+    ///
+    /// Panics if the instruction doesn't support arguments.
+    pub fn append_inst_args(&mut self, inst: Inst, new_args: &[Value]) {
```

Comment:
You could use `func.dfg[inst]` instead of `func.dfg.insts[inst]`. That way you can avoid adding this method.

Reply:
Nice, I'll drop this commit then.